.github/workflows/run-metalgateway-functional-tests.yml

```diff
@@ -0,0 +1,61 @@
+# This workflow will build a Java project with Maven, and cache/restore any dependencies to improve the workflow execution time
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+
+name: Execute functional tests for MetalGateway related workflows
+
+on:
+  workflow_dispatch:
+    inputs:
+      metro:
+        description: 'metro code where you want to deploy'
+        default: 'DA'
+        required: true
+      privateIpv4SubnetSize:
+        description: 'private ipv4 subnet size'
+        default: '16'
+        required: true
+      vxlan:
+        description: 'vxlan id'
+        default: '100'
+        required: true
+      token:
+        description: 'Users temp API KEY token (Note: delete after use)'
+        required: true
+      project_uuid:
+        description: 'Project UUID where to deploy instance'
```

Comment:
We may as well make `project_uuid` part of `secrets` too.

Reply:
Resolved in 50d355bfc1f714d8086cfa331e9c6ea35eb31ae4